### PR TITLE
Ecl mobility shortcut fix

### PR DESCRIPTION
**Pull request by andlaus**
this PR contains patches for minor issues which i've stumbled over during hacking on the blackoil solvent extension. all except the last one are quite definitely harmless.

### CODE REVIEW

**Comment by andlaus** [comment]
jenkins build this with downstreams please

**Comment by atgeirr** [comment]
The last commit requires this to be tested a little more carefully, as it changes runs. What cases have you verified (including checking that the results are equivalent)?

File: ebos/eclfluxmodule.hh [hunk 1]
```diff
@@ -26,7 +26,7 @@
  * \brief This file contains the flux module which is used for ECL problems
  *
  * This approach to fluxes is very specific to two-point flux approximation and applies
- * what the Eclipse Technical Description calls the "NEWTRAN" tramsmissibilty approach.
+ * what the Eclipse Technical Description calls the "NEWTRAN" transmissibilty approach.
```
Comment (blattms) [inline ebos/eclfluxmodule.hh]: Still one i missing in transmissibility, though.

Comment (andlaus) [inline ebos/eclfluxmodule.hh]: right, thanks. that's a whack-a-mole word for typos ;)

fix pushed.

**Comment by andlaus** [comment]
I've tested that the results for norne stay identical with virtually unchanged performance, and I've tested that model 2 still runs through. Lacking objections, I'll merge this tomorrow or on Monday. (whichever comes first. ;)

**Pull request merged**